backend/integrationTesting/integrationSuite.go

```diff
@@ -13,14 +16,21 @@ type IntegrationTestSuite struct {
 }
 
 func (suite *IntegrationTestSuite) SetupSuite() {
+	if testing.Short() {
+		suite.T().Skip()
+	}
 	/** Since the  testSetupAbstraction reuses the container parallel use of this suite would cause conflicts. --> We make sure it's not used in parallel using a mutex */
```

Comment:
```suggestion
	/** Since the testSetupAbstraction reuses the container parallel use of this suite would cause conflicts. --> We make sure it's not used in parallel using a mutex */
```